nncf/quantization/algo.py

```diff
@@ -275,10 +277,12 @@ def get_potential_quantized_modules(self, target_model: NNCFNetwork) -> List[Pot
                 graph_operation = associated_ops[0]
                 metatype = graph_operation[InsertionPointGraph.OPERATOR_METATYPE_NODE_ATTR]
                 qconfig_list = meta_vs_qconfig_map[metatype]
-                if qconfig_list is None:
+                if qconfig_list is not None and len(qconfig_list) == 0:  # Empty list = wildcard quantization
```

Comment:
You should explain what is `wildcard quantization` somewhere and how it can be properly specified. Examples are desirable.

Reply:
Already explained [here](https://github.com/openvinotoolkit/nncf_pytorch/blob/300ccd0998631df8b9be2dafa14ef0973c7518f2/nncf/hw_config.py#L185-L189). All these TODOs are for me, anyway, and I will be resolving these in the next PR.